src/main/java/roomescape/domain/ReservationTimeAvailabilities.java

```diff
@@ -0,0 +1,31 @@
+package roomescape.domain;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ReservationTimeAvailabilities {
+
+    private final Map<ReservationTime, Boolean> values;
+
+    public static ReservationTimeAvailabilities of(final List<ReservationTime> reservationTimes, final List<Reservation> reservations) {
+        final Map<ReservationTime, Boolean> reservationTimeAvailabilities = new HashMap<>();
+        reservationTimes.forEach(reservationTime -> reservationTimeAvailabilities.put(
+                reservationTime, isTimeAvailable(reservations, reservationTime)));
+
+        return new ReservationTimeAvailabilities(reservationTimeAvailabilities);
+    }
```

Comment:
```suggestion
    public static ReservationTimeAvailabilities of(final List<ReservationTime> reservationTimes, final List<Reservation> reservations) {
        final Map<ReservationTime, Boolean> reservationTimeAvailabilities = new HashMap<>();
        reservationTimes.forEach(reservationTime -> 
                reservationTimeAvailabilities.put(reservationTime, isTimeAvailable(reservations, reservationTime)));

        return new ReservationTimeAvailabilities(reservationTimeAvailabilities);
    }
```

이게 가독성이 더 좋지 않을까 싶어.

Reply:
이것도 동의합니다